src/Core/Components/Dialog/FluentDialog.razor.cs

```diff
@@ -90,10 +90,15 @@ protected override Task OnAfterRenderAsync(bool firstRender)
                 instance.FluentDialog = this;
             }
 
-            return ShowAsync();
+            var pfe = await ShowAsync();
```

Comment:
A explicit name like `focusedElement` or `previouslyFocusedElement` should be better.